lib/evss/claims_service.rb

```diff
@@ -16,15 +14,11 @@ def initialize(*args)
     end
 
     def all_claims
-      rescue_evss_errors(%w[EVSS_7021 EVSS_7022]) do
```

Comment:
Are you getting rid of these to use elsewhere? Also is that rescue_evss_errors method still being used and or necessary?

Reply:
Nope it's completely removed from the project.

Those EVSSErrors are now being handled by another sentry processor and being raised (rather than rescued) as `EVSS::ErrorMiddleware::EVSSError`